src/shared/varlink.h

```diff
@@ -86,6 +86,10 @@ int varlink_callb(Varlink *v, const char *method, JsonVariant **ret_parameters,
 int varlink_invoke(Varlink *v, const char *method, JsonVariant *parameters);
 int varlink_invokeb(Varlink *v, const char *method, ...);
 
+/* Enqueue method call, expect a reply now, and possibly more later, which are all delivered to the reply callback */
+int varlink_observe(Varlink *v, const char *method, JsonVariant *parameters);
+int varlink_observeb(Varlink *v, const char *method, ...);
```

Comment:
Those functions are not used anywhere in this PR, right?

Reply:
nope, they are not. homed makes use of them though.

I was pondering whether to post this split out or not. The reason I did it after all was that the current code claimed it supported method calls with "more" set (i.e. method calls that have multiple replies, in a "subscribe" fashion), but actually was totally broken in this regard. Now, we don't use "more" yet in mainline, but I couldn't live with the fact that the code claimed to do something it didn't...

(there are also a number of other functions in varlink.c currently not used in mainline)

Reply:
OK, let's leave this in then. No need to make complicate things too much.